server/liveblog/themes/themes.py

```diff
@@ -275,17 +275,24 @@ def publish_related_blogs(self, theme):
         # for t in self.get_children(theme['name']) + [theme['name']]:
         #     terms.append({'term': {'blog_preferences.theme': t}})
         blogs = get_resource_service('blogs').get(req=None, lookup={})
+        outputs = get_resource_service('outputs').get(req=None, lookup={})
```

Comment:
no, it shouldn't be the same as there isn't a reference to blog's pots in the blog's resource.